backend/routes/users.js

```diff
@@ -27,7 +27,7 @@ router.post('/signup', async function (req, res, next) {
                 "lastName": lastName,
                 "contact": contact,
                 "school": school,
-                "payment": payment,
+                "paymentMethods": paymentMethods ? paymentMethods : [],
```

Comment:
When users sign up, they do not have to input payment methods. So default payment methods will be none. However, they can edit their payment methods later in the details tab under their profile page. 
So I don't think we would have a paymentMethods attribute in the req.body.

Reply:
It's fine. When this field is not included in the req.body for signup request, it can still be processed. I tested it with Postman. 
![image](https://user-images.githubusercontent.com/37158862/89754184-b9c38b00-da8f-11ea-81c3-61fa18dc3b01.png)

Reply:
Feel free to modify that if you still think it might be confusing. I keep it the current way only because it makes database initialization easier and won't affect the actual functionality.

Reply:
aahh okay, I just thought we will only be using the payment attribute in the PUT request haha but this works! I'll go ahead and resolve thiss